action.yml

```diff
@@ -1,11 +1,13 @@
+---
+
 name: "Setup HashiCorp nomad-pack"
 author: "Kerim Satirli"
-description: "A GitHub Action to install a specific version of HashiCorp Packer and add it to PATH."
+description: "A GitHub Action to install a specific version of HashiCorp nomad-pack and add it to PATH."
 inputs:
   version:
     description: "The nomad-pack version to install and add to PATH. Supports semver versioning."
     required: true
-    default: "latest"
+    default: "0.0.1-techpreview2"
```

Comment:
Same parameterized version comment

Reply:
I like this idea.

One option that comes to mind: This could be solved by getting the value from a GitHub Actions Secret, so you could more easily set this value through the GH API (incl. Terraform, for example).

Upside: easier control across multiple repositories with regards to the version of `nomad-pack` to use.

Downside: no easy visibility in the code (the GHA configuration YAML) which version a specific commit was run against.

Thoughts @DerekStrickland and @joatmon08?

Reply:
Dangit... you had to add "easily" to that requirement... Yeah, I mean you can always log.  One thing that comes to mind is find a way to inject the version in into the run description? I feel like they must be doing that in CRT (see [this](https://github.com/hashicorp/nomad/actions/runs/2861288880) run).

Reply:
Personal preference - I like it non-parametrized here. It would be nice to override this with a GitHub Actions secret (I guess this is a user choice) but setting the default here is something you can verify based on commit and within the repository itself. I noticed the version is an output, so _hopefully_ that should at least surface this for each repository. It's also a good way to point out which is a "stable" and "tested" version (unless there is a plan to run acceptance tests across all `nomad-pack` versions).

Reply:
Yeah, I like this idea. I think we can use [workflow_dispatch](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) for this.

I'll test good ways of adding this, so it is additive vs. just adding an extra human step and will revisit this in a future version / PR.